### PR TITLE
Feeds

### PR DESCRIPTION
**Pull request by minaorangina**
- adds the pub sub `UPDATE_FEEDS` event to relevant endpoints
- moves creating feed item, saving feed item, getting recipients' user id and emitting `UPDATE_FEEDS` to one function
- creates function to fetch feed items (WIP #58)

Note: @heron2014 the structure of the `feeds` table might have to change a bit.  Currently I've removed the `NOT NULL` constraint for event_id [(see here)](https://github.com/DRDD2016/api/blob/b88b752c5daa4f9eeac958ef7450e6629a843421/src/db/schema.sql#L39) so that an error isn't thrown if an event is deleted

### CODE REVIEW

File: src/lib/update-feeds.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+};
+
+export default function updateFeeds (req, res, next) {
+
+  const subject_user_id = req.subject_user_id;
+  const event_id = req.event_id;
+  const informAllInvitees = req.informAllInvitees;
+  const deletingEvent = req.method === 'DELETE' && (/\/events\/\d+/).test(req.url);
+
+  if (!subject_user_id || !event_id || informAllInvitees === undefined) {
```
Comment (heron2014) [inline src/lib/update-feeds.js]: @minaorangina do we need to explicitly compare to ```undefined``` here?

File: src/lib/handlers.js [hunk 2]
```diff
@@ -37,27 +33,42 @@ import resetUserPassword from './auth/reset-user-password';
...
+  next(); // --> updateFeeds
+}
+
 export function deleteEventHandler (req, res, next) {
+  // updateFeeds happens before this step
   deleteEvent(client, req.params.event_id)
-  .then((deleted_event_id) => {
-    res.json(deleted_event_id);
+  .then(() => {
+    return res.status(204).end();
```
Comment (heron2014) [inline src/lib/handlers.js]: @minaorangina could you clarify this line for me. Here you end the request after successfully deleted event?

Comment (minaorangina) [inline src/lib/handlers.js]: @heron2014 yeah.  there's no data to send back, so just a matter of ending the request.  The client will receive a 204 and that will be the end

File: src/lib/update-feeds.js (hunk 1 above)
Comment (minaorangina) [inline src/lib/update-feeds.js]: @heron2014 yes, because it's a boolean value, i don't want `informAllInvitees === false` to be misinterpreted

Comment (heron2014) [inline src/lib/update-feeds.js]: aha!

File: src/lib/handlers.js (hunk 2 above)
Comment (heron2014) [inline src/lib/handlers.js]: ok thanks!

**Pull request merged**